### ISSUE TITLE
Should some documents be status-specific?

### ISSUE THREAD

**Issue by hughstimson** [PlaceSpeak/LUCD#8]
Linda Low makes the point that in Vancouver's existing reporting of permit applications, documents are often listed as being relevant to particular status events (e.g. there might be a planning document associated with an initial application, and then another associated with a re-application).

Should documents be associated with individual statuses? I.e. 
```
- statuses
- - status *
- - - status name *
- - - status date *
- - - document
- - - - document name
- - - - document abstract
- - - - document URL
- - - [optional additional documents]
- - [optional additional status events]
```

JSONically, that might look like
```json
    "statuses": [
        {
            "status_name": "Approved",
            "status_date": "2014-03-11",
            "documents": [
                {
                    "document_name": "Notice of Public Hearing",
                    "document_url": "http://former.vancouver.ca/commsvcs/planning/rezoning/applications/963e19th/documents/PH_postcard.pdf"
                }
            ]
        },{
            "status_name": "Re-submitted",
            "status_date": "2010-07-05",
            "documents": [
                {
                    "document_name": "Notice of Public Hearing",
                    "document_url": "http://former.vancouver.ca/commsvcs/planning/rezoning/applications/963e19th/documents/PH_postcard.pdf"
                },
                {
                    "document_name": "Notice of Revised Rezoning Application and Invitation to Community Open House",
                    "document_url": "http://former.vancouver.ca/commsvcs/planning/rezoning/applications/963e19th/documents/notificationOHApr13.pdf"
                }
            ]
        }
    ]
```

Some documents may not be connected to a particular status, which is fine, they could go in the stand-along `documents` field. What about documents that are linked unchanged to multiple statuses?

**Comment by hughstimson** [comment]
Once the standard is re-arranged into linked tables (see issue #12) this can be addressed by adding an extra, optional `Status` field in the `Documents` dataset. That's kind of a weak linkage because it assumes that the generating system will always use an indentical string to describe the status in both places, but I guess that's a safe enough assumption.

**Comment by hughstimson** [comment]
Closed by pull request #14.
